API/utils/MailROCC.php

```diff
@@ -1,7 +1,7 @@
 <?php
 require '../classes/class.phpmailer.php';
 require '../classes/class.smtp.php';
-require '../classes/class.mailsettings.php';
+//require '../../classes/class.mailsettings.php'; file doesn't exists
```

Comment:
Can you put this line back in? It's okay if it doesn't run. I'm working on adding it back in. https://github.com/techteam25/ROCC/pull/5